README.md

```diff
@@ -1,54 +1,69 @@
+</picture>
+
 <p align="center">
   <a href="https://1password.com">
-    <img alt="1Password logo" src="https://user-images.githubusercontent.com/7430639/205342015-46801fd8-6701-482f-9da9-e21e7e39b3a1.svg" height="64">
-    <h3 align="center">1Password Shell Plugins</h3>
+    <h1 align="center">1Password Shell Plugins</h1>
   </a>
 </p>
 
 <p align="center">
-  Seamless authentication for every tool in your terminal. <sup><b><a href="#-beta-notice">[BETA]</a></b></sup>
+ <h4 align="center"> Authenticate any CLI with your fingerprint </h4>
 </p>
 
 <p align="center">
-  <a href="https://developer.1password.com/docs/cli/shell-plugins"><b>Available plugins</b></a> ·
-  <a href="https://developer.1password.com/docs/cli/"><b>1Password CLI docs</b></a> ·
-  <a href="https://developer.1password.com/docs/cli/shell-plugins/contribute"><b>Contribution docs</b></a>
+  <a href="https://developer.1password.com/docs/cli/shell-plugins/"><img alt="Get Started" src="https://user-images.githubusercontent.com/45081667/226940040-16d3684b-60f4-4d95-adb2-5757a8f1bc15.png" height="37" /></a>
+  <a href="https://developer.1password.com/docs/cli/shell-plugins/contribute"><img alt="Build" src="https://user-images.githubusercontent.com/45081667/226941089-473be407-5417-48e3-8710-55b2d4ec761a.png" height="37" /></a>
 </p>
 <br/>
 
 ---
 
-No more plaintext credentials in your home directory. Automatically authenticate every CLI you use with [1Password CLI](https://developer.1password.com/docs/cli/) + [Shell Plugins](https://developer.1password.com/docs/cli/shell-plugins/). Approve temporary credential usage in your terminal with biometrics.
-
-## 🪄 Usage
+Eliminate plaintext credentials in your home directory. Automatically authenticate every CLI you use with [1Password CLI](https://developer.1password.com/docs/cli/) + [Shell Plugins](https://developer.1password.com/docs/cli/shell-plugins/). Approve temporary credential usage in your terminal with biometrics.
 
- <picture>
-   <source srcset="https://developer.1password.com/videos/aws.webp" type="image/webp" />
-   <img src="https://user-images.githubusercontent.com/7430639/205337855-41604aca-0ddb-4eab-a5f0-fb9107e09d8d.gif" alt="Example of 1Password Shell Plugins with AWS: user runs an `aws` command, a Touch ID prompt shows up, and `aws` is automatically authenticated" style="max-width: 100%; display: inline-block;" />
+## 🪄 Quick demo
+<p align="center">
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/45081667/227191964-9629476d-a49e-475d-b8cb-2115c302025d.gif">
+   <img src="https://user-images.githubusercontent.com/45081667/227197994-6fdb2cad-c240-4cb7-ba4c-77f2483606ab.gif" alt="Example of 1Password Shell Plugins with AWS: user runs an `aws` command, a Touch ID prompt shows up, and `aws` is automatically authenticated" style="max-width: 100%; display: inline-block;" />
```

Comment:
This video is very low res. Also, the shadow/glow gets truncated. I'd say only add a shadow if you can ensure a smooth transition between the shadow and the background.

<img width="283" alt="image" src="https://user-images.githubusercontent.com/7430639/227500087-89c1b2a3-9142-4b68-bc41-109970b98797.png">

Reply:
Also, realizing now: what does this video do that the current one doesn't?

Reply:
The changes introduced mainly came from interviewing people. Some folks did not like the previous video because it didn't quite look as a terminal, because it was too wide and without round edges. Also people expressed interest in the shadow. However, I didn't find a way to implement these features in MD so as a workaround I recorded a terminal that casts shadows on a white background.

Reply:
I like the idea of making improving the cosmetics to make it feel more like a terminal window, but that new version should then not make the status quo cosmetically worse by lowering the resolution and truncating the shadow.

Reply:
If we feel this needs further improvements, we could iteratively change only this gif